processor/factory.go

```diff
@@ -34,7 +34,8 @@ func createDefaultConfig() component.Config {
 		LocalSourceMapConfiguration: &LocalSourceMapConfiguration{
 			Path: ".",
 		},
-		Timeout:                       5 * time.Second,
+		Timeout:            5 * time.Second,
+		SourceMapCacheSize: 128,
```

Comment:
Do we have a sense of when this was falling over? I'm not sure what a sensible default should be.

Reply:
I would guess that it was much lower than this. I dont have hard numbers though. Ill make a ticket to see if we can add some metrics to the collector telemetry